File: hw/gudino/u1/hw07ChickenCompleatVersionInPy&JS/mainChicken.py
from chicken import Chicken

if __name__ == "__main__":
    print("This is my Chicken Farm Simulator")

    chicken2 = Chicken(0, "Maruja", "white", 1, True)

    owner = "Bryan Gudino"
    id = 1
    name = "Lucy"
    color = "black and brown"
    age = 2
    is_molting = False

    chicken = Chicken(id, name, color, age, is_molting)
    print("The chicken is ---->", chicken)

    print("chicken id --->", chicken.get_id(), chicken.get_name())
    chicken.get_id()
    chicken.do_stuff()
    chicken2.get_id()
    chicken2.do_stuff()

File: hw/gudino/u1/hw07ChickenCompleatVersionInPy&JS/mainchicken.js
import { Chicken } from "./chicken.js";

console.log("This is my Chicken Farm Simulator");

const chicken2 = new Chicken(0, "Maruja", "white", 1, true);

let owner = "Bryan Gudino";
let id = 1;
let name = "Lucy";
let color = "black and brown";
let age = 2;
let isMolting = false;

const chicken = new Chicken(id, name, color, age, isMolting);
console.log("The chicken is ---->", chicken.toString());

console.log("chicken id --->", chicken.getId(), chicken.getName());
chicken.getId();
chicken.doStuff();
chicken2.getId();
chicken2.doStuff();
